Snippets/ahc_json.swift

```diff
@@ -0,0 +1,71 @@
+// snippet.hide
```

Comment:
just a heads up, these are likely to run into https://github.com/tayloraswift/swift-unidoc/issues/265

Reply:
this problem was fixed in https://github.com/tayloraswift/swift-unidoc/pull/266